docker-compose.yml

```diff
@@ -9,7 +9,9 @@ services:
       - 3000:5000
     volumes:
       - ./config.local.yml:/glider-dac/config.local.yml
-      # - .:/glider-dac/  # For local development
+    #  # - .:/glider-dac/  # For local development
+      - ./users.db:/glider-dac/users.db
```

Comment:
This wasn't intended to be included, but may want to anyhow.  If not, we can just squash down.